JASP-Desktop/po/jasp_nl.po

```diff
@@ -16078,10 +15845,2443 @@ msgctxt "RibbonModel|"
+msgstr "McDonald's ω"
+
+msgctxt "ReliabilityBayesian|"
+msgid "Posterior predictive check"
+msgstr "Posterior voorspellende verificatie"
+
+msgctxt "ReliabilityBayesian|"
+msgid "Cronbach's α"
+msgstr "Cronbach's α"
+
+msgctxt "ReliabilityBayesian|"
+msgid "Guttman's λ2"
+msgstr "Guttman's λ2"
+
+msgctxt "ReliabilityBayesian|"
+msgid "Guttman's λ6"
+msgstr "Guttman's λ6"
+
+msgctxt "ReliabilityBayesian|"
+msgid "Greatest lower bound"
+msgstr "Grootste ondergrens"
+
+msgctxt "ReliabilityBayesian|"
+msgid "Average interitem correlation"
+msgstr "Gemiddelde inter-item correlatie"
+
+msgctxt "ReliabilityBayesian|"
+msgid "Mean"
+msgstr "Gemiddelde"
+
+msgctxt "ReliabilityBayesian|"
+msgid "Standard deviation"
+msgstr "Standaardafwijking"
+
+msgctxt "ReliabilityBayesian|"
+msgid "Individual Item Statistics"
+msgstr "Individuele Itemstatistieken"
+
+msgctxt "ReliabilityBayesian|"
+msgid "McDonald's ω  (if item dropped)"
+msgstr "McDonald's ω  (als item is verwijderd)"
+
+#, fuzzy
+msgctxt "ReliabilityBayesian|"
+msgid "Cronbach's α (if item dropped)"
+msgstr "Cronbach's α (als item is verwijderd)"
+
+msgctxt "ReliabilityBayesian|"
+msgid "Guttman's λ2 (if item dropped)"
+msgstr "Guttman's λ2 (als item is verwijderd)"
+
+msgctxt "ReliabilityBayesian|"
+msgid "Guttman's λ6 (if item dropped)"
+msgstr "Guttman's λ6 (als item is verwijderd)"
+
+msgctxt "ReliabilityBayesian|"
+msgid "Greatest lower bound (if item dropped)"
+msgstr "Grootste ondergrens (als item is verwijderd)"
+
+msgctxt "ReliabilityBayesian|"
+msgid "If item dropped plot"
+msgstr "Als item is verwijderd"
+
+msgctxt "ReliabilityBayesian|"
+msgid "Order items"
+msgstr "Rangschik items"
+
+msgctxt "ReliabilityBayesian|"
+msgid "Order items by mean"
+msgstr "Rangschik items naar gemiddelde"
+
+msgctxt "ReliabilityBayesian|"
+msgid "Order items by KL-divergence"
+msgstr "Rangschik items naar KL-divergentie"
+
+msgctxt "ReliabilityBayesian|"
+msgid "Order items by KS-distance"
+msgstr "Rangschik items naar KS-afstand"
+
+msgctxt "ReliabilityBayesian|"
+msgid "Item-rest correlation"
+msgstr "Item-rest correlatie"
+
+msgctxt "ReliabilityBayesian|"
+msgid "Plot Posteriors"
+msgstr "Grafiek Posteriors"
+
+msgctxt "ReliabilityBayesian|"
+msgid "Fix range to 0-1"
+msgstr "Zet bereik vast op 0-1"
+
+msgctxt "ReliabilityBayesian|"
+msgid "Display Priors"
+msgstr "Toon Priors"
+
+msgctxt "ReliabilityBayesian|"
+msgid "Probability for: "
+msgstr "Kans op: "
+
+msgctxt "ReliabilityBayesian|"
+msgid "< Reliability <"
+msgstr "< Betrouwbaarheid <"
+
+msgctxt "ReliabilityBayesian|"
+msgid "Shade posterior region in plot"
+msgstr "Markeer posterior gebied in grafiek"
+
+msgctxt "ReliabilityBayesian|"
+msgid "Convergence"
+msgstr "Convergentie"
+
+msgctxt "ReliabilityBayesian|"
+msgid "MCMC parameters"
+msgstr "MCMC parameters"
+
+msgctxt "ReliabilityBayesian|"
+msgid "No. samples"
+msgstr "Aantal steekproeven"
+
+msgctxt "ReliabilityBayesian|"
+msgid "No. burnin samples"
+msgstr "Aantal opwarmrondes"
```

Comment:
Aantal opwarmmonsters

of

Opwarmsteekproefgrootte

... samples niet rondes, maar monsters klinkt zo raar